tests/go/boilerplate_test.go

```diff
@@ -0,0 +1,49 @@
+package _go
```

Comment:
What was the though process behind using `go` as the package name and not `main`? Does the `_` have special meaning?

Reply:
wasn't aware of this, that's a screw up from the IDE. Fixed